--- ./lua/lazy/help.lua	1969-12-31 18:00:01.000000000 -0600
+++ help.lua	2023-09-05 22:13:54.833884328 -0500
@@ -39,7 +39,7 @@
 
 function M.update()
   if Config.plugins["lazy.nvim"] then
-    vim.cmd.helptags(Config.plugins["lazy.nvim"].dir .. "/doc")
+    vim.cmd.helptags(Config.options.help.root .. "/doc" or Config.plugins["lazy.nvim"].dir .. "/doc")
   end
   if Config.options.readme.enabled == false then
     return
